### ISSUE TITLE
Functional Accessibility Testing

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#718]
Contact Scott Williams to have him conduct a functional accessibility test on production Fulcrum.

- [x] deploy a master branch to staging and preview that includes the Leaflet updates in #747
- [x] create google doc with links to URLs/examples for each of the items in the list below.

https://docs.google.com/document/d/1n34mT1FvUGmjQzDquXUBrG3lwjP-2shdDm1kJCNe9uU/edit

**Comment by jmcglone** [comment]
@mbakeryo feel free to add to the list above if you can think of things Scott should be testing in the UI

**Comment by jmcglone** [comment]
@mbakeryo when you get a chance can you take a look at the google doc I'll send Scott?

https://docs.google.com/document/d/1n34mT1FvUGmjQzDquXUBrG3lwjP-2shdDm1kJCNe9uU/edit

**Comment by mbakeryo** [comment]
@jmcglone I added page navigation to the Book Page section. Feel free to remove if you disagree. Also, will we be asking Scott to test out/review Search within a book?

**Comment by jmcglone** [comment]
E-mailed Scott on 2/8

**Comment by jmcglone** [comment]
Will be reviewed in March

**Comment by jmcglone** [comment]
E-mailed Scott on March 14 for a status update and awaiting response

**Comment by mbakeryo** [comment]
@jmcglone Hi! Any word from Scott?

**Comment by jmcglone** [comment]
@mbakeryo I e-mailed again on March 29 – no response yet.

**Comment by jmcglone** [comment]
E-mailed Scott on April 28

**Comment by mbakeryo** [comment]
Unblocking this ticket, since we're no longer going to do testing through Scott. Will pursue using the Deque tool and reaching out to Ben Howell to run accessibility tests.

**Comment by jmcglone** [comment]
Will be done in January 2018 by MSU UARC
